frontend/src/store/modules/search/mutations.js

```diff
@@ -6,15 +6,19 @@ export default {
             latitude: searchCity.center[1],
             fullName: searchCity.place_name
         };
+        state.isLoading = true;
```

Comment:
no, commit(SET_SEARCH_CITY, true) must be executed after commit(SET_SEARCH_CITY, ...)